react-toggle-switch-component/src/toggle-switch.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+class ToggleSwitch extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
While this does work, I would remove everything from state except the `isToggled` property. The reason for this is that you always want to store in state only what is necessary for the ui to be properly updated. You can set the appropriate classnames and such within the handleToggle function.